tests/test_parse.py

```diff
@@ -201,19 +202,17 @@ def test_process_results(self):
                       'alg_length': 900}}]
         }
 
-        out_results = process_results([0.80], [50], [0.30], [30], best_hits)
+        out_results = process_results([0.80], [50], [0.30], [30], best_hits,
+                                      self.base)
+        # removing the sumout_resultsmary pointer so we don't need to test
```

Comment:
It used to be an array where each element had the values that now are being written directly to a file. Now, it's a pinter to an open file, thus not testing.